src/sidebar/components/autocomplete-list.js

```diff
@@ -4,6 +4,23 @@ import { useMemo } from 'preact/hooks';
 import propTypes from 'prop-types';
 
 const defaultListFormatter = item => item;
+/**
+ * @typedef AutocompleteListProps
+ * @prop {number} [activeItem] - The index of the highlighted item.
+ * @prop {string} [id] - Optional unique HTML attribute id. This can be used
+ *   for parent `aria-controls` coupling.
+ * @prop {string} [itemPrefixId] - Optional unique HTML attribute id prefix
+ *   for each item in the list. The final value of each items' id is
+ *   `{itemPrefixId}{activeItem}`
+ * @prop {(Object|string)[]} list - The list of items to render. This can be a simple
+ *   list of strings or a list of objects when used with listFormatter.
+ * @prop {(a: Object|string, b?: number) => any} [listFormatter] - An optional formatter
```

Comment:
The meaning of `a` and `b` wasn't immediately obvious here. Can you use more specific names for them.

Reply:
indeed!